pymilvus/orm/constants.py

```diff
@@ -27,6 +27,7 @@
 
 OFFSET = "offset"
 LIMIT = "limit"
```

Comment:
Could remove the LIMIT relate constants since not inuse

Reply:
currently, we still use query interface of milvus client, so internally, limit is still used, the 'batch_size' parameter is used to avoid ambiguity of meaning